### DISCUSSION TITLE
Contributing to the project

### DISCUSSION THREAD

**Discussion by aash96** [fineanmol/Hacktoberfest2021#1355] [Q&A] [answered]
Hi @fineanmol !
Just a couple of clarifications, 
1.I can update my name in the contributors.md after the 1st October as a single pull request right?(because updating now doesn't count as pull request..)
2.I dont need to get an issue assigned to me right? And while creating the pull request it doesn't matter which issue i'm working on right?
Thanks.

**Comment by fineanmol** [comment, marked as answer]
> Hi @fineanmol ! Just a couple of clarifications, 1.I can update my name in the contributors.md after the 1st October as a single pull request right?(because updating now doesn't count as pull request..) 2.I dont need to get an issue assigned to me right? And while creating the pull request it doesn't matter which issue i'm working on right? Thanks.

1. Yes No Problem, Just rebase the repo on 1st and make a fresh PR on that day so I can merge it without conflicts. 
2. You don't need to get an issue assign first, you can work on any issue. Just make a fresh pull request and Star ⭐️ the repo to earn "**_hacktober-accepted_**" label for the event.
